stylus/cozy-ui/defaults.styl

```diff
@@ -42,6 +23,34 @@ body
     @extend $font-labor
 
 
+// UTILS
+@import '../ui-base/mixins'
+:global
+    .coz-hidden
+        @extend $hidden
+
+    .coz-desktop
+        @media (max-width: (1023/basefont)rem)
+            display none !important
```

Comment:
Hah I see. It's not really their fault but the fact that their entire selector must have one or more class selector in it. So in this case it totally makes sense, you're right, keep the `!important`. 
You won! 🤘